<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Equipement</class>
 <widget class="QMainWindow" name="Equipement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>941</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Machine</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>531</width>
        <height>411</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ajouter/modifier</string>
       </attribute>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>340</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>300</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>50</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>90</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>130</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>170</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>50</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>90</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>130</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>170</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>90</y>
          <width>351</width>
          <height>281</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>40</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Trier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>40</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>10</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>40</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Actualiser</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>10</y>
          <width>73</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>id</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>description</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>nom</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="comboBox_2">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>10</y>
          <width>73</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>id</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>nom</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>description</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>100</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_11">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>190</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_13">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>250</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Exporter Excel</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Statistiques</string>
       </attribute>
       <widget class="QFrame" name="horizontalFrame">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>481</width>
          <height>391</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:transparent;</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>tache</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>531</width>
        <height>411</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Ajouter/modifier</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>340</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>300</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_7">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>50</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_8">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>90</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_9">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>130</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_10">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>170</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>50</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>90</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>130</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>170</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>130</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Heure fin</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>50</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>90</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Heure debut</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>170</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>id machine</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_13">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>90</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_14">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>160</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_15">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>130</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEdit">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>50</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>20</y>
          <width>401</width>
          <height>281</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_10">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>310</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Export PDF</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_9">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>310</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_12">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>340</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>120</y>
        <width>361</width>
        <height>361</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1035</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
